app/services/user/bulk_destroy.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class User::BulkDestroy
+  include Service::Base
+
+  params do
+    attribute :user_ids, :array
+
+    validates :user_ids, length: { maximum: 100 }
+  end
+
+  model :users
+  policy :can_delete_users
+  step :delete
+
+  private
+
+  def fetch_users(params:)
+    ids = params.user_ids.to_a
```

Comment:
Why do you need to call to_a here since the attribute is already an :array ?

Reply:
This was copied from an existing service 😅 I've removed it and it seems to work